.env_sample

```diff
@@ -1,2 +1,7 @@
-host=
-port=
\ No newline at end of file
+dbuser=
+dbpw=
+dbhost=
+dbport=
+database=
+serverhost=
+port=
```

Comment:
This is quite trivial, but it seems to be the convention to declare these variable names in all caps (as seen in the dotnev documentation). Just fyi!

Reply:
Thanks for pointing this out. I'll be updating them!